docs/index.md

```diff
@@ -11,4 +11,5 @@ download_files
 sce_file_contents
 getting_started
 faq
+citation
```

Comment:
I think thinks makes sense as a location. I could see an argument for before FAQ, but I do prefer this.